EDITORIAL_GUIDELINES.md

```diff
@@ -0,0 +1,81 @@
+   - **Example**: **MUST**, **SHOULD**, **MAY**
+
+5. **Descriptions and Content Constraints**
+   - **Style**: Normal text, but ensure clear headings and subheadings
+   - **Reasoning**: These sections should remain easily readable, with clear headings for each sub-section to facilitate navigation.
+   - **Example**:
+     - **Description**: A name assigned to a grouping of resources or services, often used to manage access and/or cost.
+     - **Content Constraints**:
+       - **Column type**: Dimension
+       - **Feature level**: Conditional
+
+6. **Attribute Names and IDs in Specifications**
+   - **Style**: Attribute Names in **`Bold`**, Attribute IDs in *`Italic`*
+   - **Reasoning**: This approach helps in visually distinguishing between attribute names and their respective IDs.
+   - **Example**:
+     - **Attribute Name**: **`Currency Code Format`**
+     - **Attribute ID**: *`CurrencyCodeFormat`*
+
+7. **Key-Value Format**
+   - **Style**: <code>Monospaced font</code>
+   - **Reasoning**: Using a monospaced font for key-value pairs ensures clarity and readability, especially for JSON or code snippets.
+   - **Example**:
+     ```json
+     {
+       "key1": "value1",
+       "key2": true,
+       "key3": 123
+     }
+     ```
+
+8. **Tables and Lists**
+   - **Style**: Use standard table formatting with clear column headings in bold.
+   - **Reasoning**: Standard table formatting with bold headings improves readability and ensures that data is well-organized and easy to interpret.
+   - **Example**:
+
+     | **Constraint**      | **Value**            |
+     |---------------------|----------------------|
+     | Column type         | Dimension            |
+     | Feature level       | Conditional          |
+     | Allows nulls        | True                 |
+     | Data type           | String               |
+     | Value format        | Not specified        |
+
+9. **Highlighting Important Paragraphs**
+
+    - **Style**: Markdown quote (>) symbol
+    - **Reasoning**: Highlighting key paragraphs ensures they capture the reader's attention, emphasizing their importance within the document.
+    - *Example*:
+
+        > Important: Ensure all critical parameters are verified.
+
+10. **Formatting Normative Statements as Bullet Lists**
+
+    - **Style**: Bullet list format
+    - **Reasoning**: Presenting normative statements in a bullet list format ensures clarity and facilitates quick scanning and comprehension.
+    - **Example**:
+    **ConsumedQuantity** and **ConsumedUnit**:
+        * MUST NOT be null if ChargeCategory is "Usage", unless ChargeClass is "Correction" or CommitmentStatus is 'Unused'
```

Comment:
Should we clarify how specified values should be formatted? In this line, it had "Correction" in double quotes but 'Unused' in single quotes before I changed both to double quote for consistency.